examples/version-migration/schema-upgrade/src/modelVersion1/appModel.ts

```diff
@@ -25,7 +25,6 @@ export class InventoryListAppModel implements IInventoryListAppModel, IMigratabl
 
 	public constructor(
 		public readonly inventoryList: IInventoryList,
-		public readonly migrationTool: IMigrationTool,
```

Comment:
The app model also gets to become ignorant of the migration tool - now it's only responsible for providing import/export, version, and dispose.